backend/controllers/otpController.js

```diff
@@ -0,0 +1,83 @@
+const nodemailer = require("nodemailer");
+const Brand = require("../model/brands");
+const OTP = require("../model/otp");
+const dbConfig = require("../config/dbConfig");
+
+const transporter = nodemailer.createTransport({
+  host: dbConfig.SMTP_HOST,
+  port: 465,
+  secure: true,
+  auth: {
+    user: dbConfig.OTP_MAIL,
+    pass: dbConfig.PASSWORD,
+  },
+});
+function generateOTP() {
+  return Math.floor(100000 + Math.random() * 900000).toString();
+}
+
+const sendOTP = async (req, res) => {
+  const { email } = req.body;
+
+  // Check if email exists
+  //Uncomment this only after integrating the brand with database
```

Comment:
i hope you uncomment it in your upcoming pr containing the fix for saving brands data in db :)

Reply:
yes, For that purpose Only I mentioned it as uncomment after integration of db.